core/src/main/java/org/apache/calcite/avatica/util/AbstractCursor.java

```diff
@@ -150,26 +151,34 @@ protected Accessor createAccessor(ColumnMetaData columnMetaData,
         throw new AssertionError("bad " + columnMetaData.type.rep);
       }
     case Types.TIME:
+      // TIME WITH LOCAL TIME ZONE is a standard ISO type without proper JDBC support.
+      // It represents a global instant in time, as opposed to local clock parameters.
+      boolean fixedInstant =
```

Comment:
'fixedInstant' is a tautology. Call it 'instant' (which is Joda time terminology)